source/features/safer-destructive-actions.css

```diff
@@ -1,7 +1,7 @@
 /* Move "close issue" and "cancel" buttons on authoring comments to the left */
 
 /* ...in issue comment form */
-/* the legacy way (kept for GitHub Enterprise) */
+/* the legacy way (kept for GHE) */
```

Comment:
GHE comments should include a date or issue or we should mark ALL of them with the expiring TODOs feature:

```js
// TODO[2021-10-01]: GHE #1234
```

My guess is that this specified piece of code is at least 2 years old and can be dropped though

Reply:
I am going to drop it when we drop refresh.

Will be much easier to drop it all at once IMO

Reply:
This really is 2 years old though, this rule can be deleted now

<img width="559" alt="Screen Shot 2021-04-07 at 12 30 49" src="https://user-images.githubusercontent.com/1402241/113815306-2452d500-979d-11eb-97c2-62544e0a2826.png">

Reply:
What about 

https://github.com/sindresorhus/refined-github/blob/edf6b1a50f1cdb55c3303869537f105b2144afa2/source/features/more-dropdown.css#L1-L7

Also 2 years old